task/sql-tasks.js

```diff
@@ -177,7 +274,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT

```

Comment:
1) use count when you need to count recrods to show intent explicitly

2) it's better to write this query like this

```
select
(select count()...) as 'T'
(select count()...) as 'TT'
```

notice from-clause is optional